cluster/aws/util.sh

```diff
@@ -1096,8 +1184,13 @@ function wait-master() {
   echo "  up."
   echo
 
+  local url="https://${KUBE_MASTER_IP}/healthz"
+  if [[ -n "${KUBE_SERVER:-}" ]]; then
+    url="${KUBE_SERVER}/healthz"
+  fi
+
```

Comment:
nit: Sins of the past, but can we rationailze the seemingly semi-redundant  KUBE_MASTER_IP, KUBE_SERVER, API_SERVERS, KUBE_MASTE_DNS_NAME ?   Or at least clearly document what each one means, and how they are different.